src/autocomplete.js

```diff
@@ -101,7 +101,7 @@ export default class Autocomplete {
 
   onResultsMouseDown() {
     this.mouseDown = true
-    this.results.addEventListener('mouseup', () => (this.mouseDown = false), {once: true})
+    setTimeout(() => (this.mouseDown = false), 100)
```

Comment:
Because we can't guarantee the `mouseup` event will happen on the result list, another fix would be to listen on document. That relies on any intermediate `mouseup` handlers not stopping event propagation up the tree, though.